### ISSUE TITLE
Engine sound in bumper camera

### ISSUE THREAD

**Issue by Ratizka** [Brawltendo/NFS-Rivals-Framerate-Unlocker#7]
The engine sound is crackling in the bumper camera. In cops Hennessey Venom GT it only crackles with no engine sound. In racer side on Ferrari F12 crackling sound is heard on the background of the engine sound. I did not test other cars.

**Comment by geekycow** [comment]
Yep, experiencing the same engine sound problem, with with more general sound break up as additional sounds get involved too.

**Comment by geekycow** [comment]
This actually happens with the base game without the mods if you use the FPS options, so it seems to be a game bug with some sounds with higher FPSs.
